libraries/botbuilder-core/botbuilder/core/teams/teams_info.py

```diff
@@ -14,6 +15,37 @@
 
 
 class TeamsInfo:
+    @staticmethod
+    async def send_message_to_teams_channel(
+        turn_context: TurnContext, activity: Activity, teams_channel_id: str
+    ) -> tuple:
```

Comment:
could you please import Tuple from typing and specify the expected types of the tuple elements